op-node/rollup/derive/batches.go

```diff
@@ -289,5 +323,47 @@ func checkSpanBatch(cfg *rollup.Config, log log.Logger, l1Blocks []eth.L1BlockRe
 			}
 		}
 	}
+
+	// Check overlapped blocks
+	if batch.GetTimestamp() < nextTimestamp {
+		for i := uint64(0); i < l2SafeHead.Number-parentNum; i++ {
+			safeBlockNum := parentNum + i + 1
+			safeBlockPayload, err := l2Fetcher.PayloadByNumber(ctx, safeBlockNum)
+			if err != nil {
+				log.Error("failed to fetch L2 block payload", "number", parentNum, "err", err)
+				// unable to validate the batch for now. retry later.
+				return BatchUndecided
+			}
+			safeBlockTxs := safeBlockPayload.Transactions
+			batchTxs := batch.GetBlockTransactions(int(i))
+			// execution payload has L1 info deposit TX, but batch does not.
```

Comment:
Are there deposit transactions which are not L1 attribute deposit transaction in execution payload?